test/csuite/timestamp_abort/main.c

```diff
@@ -549,14 +553,14 @@ thread_run(void *arg)
          * timestamp transaction, not before, because of the possibility of rollback in the
          * transaction. The local table must stay in sync with the other tables.
          */
-        data.size = __wt_random(&rnd) % MAX_VAL;
+        data.size = __wt_random(&td->data_rnd) % MAX_VAL;
         data.data = lbuf;
         cur_local->set_value(cur_local, &data);
         testutil_check(cur_local->insert(cur_local));
 
         /* Save the timestamps and key separately for checking later. */
         if (fprintf(fp, "%" PRIu64 " %" PRIu64 " %" PRIu64 "\n", active_ts,
-              durable_ahead_commit ? active_ts + 4 : active_ts, i) < 0)
+              durable_ahead_commit ? active_ts + 1 : active_ts, i) < 0)
```

Comment:
This doesn't look like where we use the third timestamp. Where is that? Maybe it just didn't change in this PR. Maybe add a comment wherever that is that says "Optionally use our third reserved timestamp"

Reply:
Yes, this is where the third timestamp is used.  active_ts had already been incremented earlier in the loop (that was unchanged).

Reply:
I've updated the three places that reserved timestamps are first used with comments.